src/azure-cli/HISTORY.rst

```diff
@@ -25,6 +25,11 @@ Release History
 * Fix `vm create` failure in Azure Stack profile.
 * vm monitor metrics tail/list-definitions: support query metric and list definitions for a vm.
 
+**SQL**
```

Comment:
Please place the release note at the top of history.rst.

Reply:
Done!